Cargo.toml

```diff
@@ -99,17 +98,16 @@ windows = { version = "0.39.0", features = [
   "Win32_UI_Input_KeyboardAndMouse",
   "Win32_UI_WindowsAndMessaging",
 ] }
-simplelog = { version = "0.12.0", optional = true }
+tracing = "0.1"
 
 [dev-dependencies]
-simplelog = "0.12.0"
+# tracing-subscriber = "0.3"
```

Comment:
I'd remove the `dev-dependencies` table entirely if we don't use any of its values; but we may need this anyway (see other comments)